akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/AccountExampleOptionState.scala

```diff
@@ -0,0 +1,103 @@
+package docs.akka.persistence.typed
+
+import akka.actor.typed.Behavior
+import akka.persistence.typed.scaladsl.{Effect, PersistentBehaviors6}
+import akka.persistence.typed.scaladsl.PersistentBehaviors6.HandlerFactoryOption
+import docs.akka.persistence.typed.AccountExampleOptionStateWithFactory.AccountEntity.{CommandHandler, EventHandler}
+
+class AccountExampleOptionState {
```

Comment:
This example is taking the same approach as AccountExample6. Boilerplate is removed with a type alias for Effect and `applyCommand` and `applyEvent` methods in Account trait.